### DISCUSSION TITLE
Disable automatic numbering

### DISCUSSION THREAD

**Discussion by olof-dev** [KaTeX/KaTeX#3180] [Q&A] [answered]
Hello! Thanks for the awesome KaTeX!

Does anyone know if it's possible to disable automatic equation numbering for all environments (like `align`, `equation` etc.)? I know one can use `align*` and so on, but in my use-case equation numbering doesn't make sense, so I'd like it disabled globally.

Thanks!

**Comment by edemaine** [comment, marked as answer]
One approach would be to add the following CSS to hide the equation numbering:
```css
.eqn-num { display: none !important; }
```

Unfortunately, I don't know how to do it at the TeX level until we support environment definition (#977).

We've also discussed having an option that configures different forms of automatic numbering, which could have an "off" setting, but it doesn't exist yet.

**Reply by olof-dev** [reply]
That's great, thank you!
